packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -287,20 +340,173 @@ export class AtomicNumericFacet implements InitializableComponent {
   }
 
   public render() {
-    if (!this.numericFacetCommon) {
-      return (
-        <FacetPlaceholder
-          numberOfValues={this.numberOfValues}
-          isCollapsed={this.isCollapsed}
-        ></FacetPlaceholder>
-      );
+    const {
+      searchStatusState: {firstSearchExecuted, hasError},
+      bindings: {i18n},
+    } = this;
+    return (
+      <FacetGuard
+        enabled={this.enabled}
+        firstSearchExecuted={firstSearchExecuted}
+        hasError={hasError}
+        hasResults={this.shouldRenderFacet}
+      >
+        {firstSearchExecuted ? (
+          <FacetContainer>
+            <FacetHeader
+              i18n={i18n}
+              label={this.label}
+              onClearFilters={() => {
+                this.focusTarget.focusAfterSearch();
+                if (this.filterState?.range) {
+                  this.filter?.clear();
+                  return;
+                }
+                this.facetForRange?.deselectAll();
+              }}
+              numberOfActiveValues={this.numberOfSelectedValues}
+              isCollapsed={this.isCollapsed}
+              headingLevel={this.headingLevel}
+              onToggleCollapse={() => (this.isCollapsed = !this.isCollapsed)}
+              headerRef={(el) => this.focusTarget.setTarget(el)}
+            />
+            {!this.isCollapsed && [
+              this.shouldRenderValues && this.renderValues(),
+              this.shouldRenderInput && (
+                <atomic-facet-number-input
+                  type={this.withInput!}
+                  bindings={this.bindings}
+                  label={this.label}
+                  filter={this.filter!}
+                  filterState={this.filter!.state}
+                ></atomic-facet-number-input>
+              ),
+            ]}
+          </FacetContainer>
+        ) : (
+          <FacetPlaceholder
+            isCollapsed={this.isCollapsed}
+            numberOfValues={this.numberOfValues}
+          />
+        )}
```

Comment:
I noticed that in atomic-result-list, instead a ternary, we have two guards (ItemDisplayGuard and ResultsPlaceholdersGuard) to control whether the placeholder of the result is displayed. 

Are we using the ternary here because we have a simple condition or is there a deeper reason?

Reply:
Hummm for result list, it's a bit more complex because of different list layouts.